internal/builtins/pkg_context.go

```diff
@@ -131,3 +131,18 @@ data: {}
 	})
 	return cm, nil
 }
+
+// DummyPkgContext returns content for package context that contains
+// placeholder value for package name. This will be used to create
+// abstract blueprints.
+func DummyPkgContext() string {
```

Comment:
WDYT about changing the name to `AbstractPkgContext` so the name is more clear about what it is?

Reply:
Yes, I like your suggestion. Done.

Reply:
This is resolved in another discussion.